templates/modcontrol/spamqueue/journal.html

```diff
@@ -0,0 +1,40 @@
+$def with (query)
+$:{RENDER("common/stage_title.html", ["Journal Spam Queue", "Moderator Control Panel", "/modcontrol"])}
+
+<div id="spam_queue_content" class="content">
+  $if query:
+    <table>
+      <thead>
+        <tr>
+          <th>User</th>
+          <th>Journal Content</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        $for user, journal_id, title, content in query:
+          <tr>
+            <td>${user} - <a href="/staffnotes/${user}">Staff notes</a></td>
+            <td>
+              <b>Journal detail page: <a href="/journal/${journal_id}?anyway=true">View journal</a></b>
+              <br /><br />
+              <b>Journal Title</b>: ${title}
+              <br /><br />
+              <b>Journal Body</b>:
+              <br /><br />
+              ${content}
+            </td>
+            <td>
+              <form action="/modcontrol/spamqueue/journal" method="post">
+                $:{CSRF()}
+                <button class="button positive" name="directive" value="${journal_id},approve">Approve</button>
```

Comment:
Would separating the journal id and action into two different fields make more sense here?

```html
<input type="hidden" name="journalid" value="${journal_id}">
<button class="button positive" name="directive" value="approve">Approve</button>
```